### PR TITLE
Change README

### PR DESCRIPTION
**Pull request by Winkel2019**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,9 @@
-# Getting Started with Create React App
+# All Movies
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+An open source project that is for all movie lovers.
 
-## Available Scripts
+## Tech
```
Comment (AdrianValdes) [inline README.md]: Review this capital T here, please.

**Review by AdrianValdes — CHANGES_REQUESTED** [review]
hi @Winkel2019 , it looks good. I had just a small request. Could you have a look at  it?

File: README.md (hunk 1 above)
Comment (Winkel2019) [reply README.md]: I agree, that's good.

**Review by AdrianValdes — APPROVED** [review]
Looks great Joyce. Merge this one in!

**Pull request merged**